sql/insert.go

```diff
@@ -53,6 +55,17 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 		colIDtoRowIndex[c.ID] = i
 	}
 
+	// Add any column not already present that has a DEFAULT expression.
+	for _, col := range tableDesc.Columns {
+		if _, ok := colIDtoRowIndex[col.ID]; ok {
+			continue
+		}
+		if col.DefaultExpr != "" {
```

Comment:
I think `DefaultExpr` is going to have to be nullable - it seems impossible to have an empty string default with this representation.